src/utils/cache.rs

```diff
@@ -85,6 +85,22 @@ pub trait CacheExt: Cache {
             }
         }
     }
+
+    fn cache_string(
```

Comment:
This method is pretty meaningless, right? ... we will never need something like this again (it creates an empty string and caches it). Maybe it should at least take a function to create a meaninfull string (which could still be empty for the health case). And can't you use `cache_resp` with a `string`?

Reply:
🤦  don't know what I was thinking with that method.